validator/src/validator.rs

```diff
@@ -892,6 +898,17 @@ where
             }
         }
 
+        // Check if DHT is bootstrapped and we can publish our record
+        while let Poll::Ready(Some(result)) = self.network_event_rx.poll_next_unpin(cx) {
+            match result {
+                Ok(NetworkEvent::DhtBootstrapped) => {
+                    self.publish_dht();
+                }
+                Ok(_) => {}
+                Err(_) => break,
```

Comment:
What does this case mean? Is this safe to just ignore? Will this recover?

Reply:
That error is thrown by the broadcast stream in the following case:

> The receiver lagged too far behind. Attempting to receive again will return the oldest message still retained by the channel.

Thus, it would be better to continue the loop instead of breaking (although that would have also continued in the next poll). I adjusted the code accordingly.